Core/DISET/private/Service.py

```diff
@@ -599,7 +598,17 @@ def _mbConnect(self, trid, handlerObj=None):
 
   def _executeAction(self, trid, proposalTuple, handlerObj):
     try:
-      return handlerObj._rh_executeAction(proposalTuple)
+      response = handlerObj._rh_executeAction(proposalTuple)
```

Comment:
I am not sure it will work as it is now. response can be S_ERROR and you will not have response[1]